### DISCUSSION TITLE
Problem with charts in templates

### DISCUSSION THREAD

**Discussion by Aidal** [ClosedXML/ClosedXML#2262] [General]
I'm working with Excel files via the ClosedXml library and have run into an issue I can't quite resolve. Normally there are no issues opening Excel files and adding data to them, formatting them etc. but I have run into an issue with a document that has a tab on which there, among other things, is a chart.

ClosedXml claims that an element has an unexpected child element at a certain path in an xml document (chart1.xml) which is part of the Excel document if opened with a zip application.

Is there a way to make ClosedXml ignore charts and similar elements all together?

I know that there is a flag that makes ClosedXml not try to refresh/validate/update formulas - maybe there is a similar flag for this or some workaround?

I'm not at all interested in touching this chart element or the tap that it is on for that matter - so how can I make ClosedXml ignore this chart and whatever it does?

**Comment by jahav** [comment]
There isn't a way to make ClosedXML ignore charts or other parts.
